metadata-ingestion/src/datahub/api/graphql/operation.py

```diff
@@ -0,0 +1,140 @@
+import logging
+from typing import Any, Dict, List, Optional
+
+from gql import gql
+
+from datahub.api.graphql.base import BaseApi
+
+logger = logging.getLogger(__name__)
+
+
+class Operation(BaseApi):
```

Comment:
I was half expecting these to be auto-generated! Is there any GraphQL Client library we can use here?